editor/plugins/skeleton_3d_editor_plugin.cpp

```diff
@@ -1210,6 +1210,7 @@ void fragment() {
 	EDITOR_DEF_RST("editors/3d_gizmos/gizmo_colors/skeleton", Color(1, 0.8, 0.4));
 	EDITOR_DEF_RST("editors/3d_gizmos/gizmo_colors/selected_bone", Color(0.8, 0.3, 0.0));
 	EDITOR_DEF("editors/3d_gizmos/gizmo_settings/bone_axis_length", (float)0.1);
+	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::FLOAT, "editors/3d_gizmos/gizmo_settings/bone_axis_length", PROPERTY_HINT_RANGE, "0.01,0.5,0.001,or_greater"));
```

Comment:
_May_ be in meters? Likely not but just asking to be sure. https://github.com/godotengine/godot/blob/652438a3952ae13e12dd4fd40756311fde81a791/editor/plugins/skeleton_3d_editor_plugin.cpp#L1495